scene/animation/tween.cpp

```diff
@@ -108,8 +108,8 @@ bool Tween::_set(const StringName& p_name, const Variant& p_value) {
 
 	String name=p_name;
 
-	if (name=="playback/speed" || name=="speed") { //bw compatibility
-		set_speed(p_value);
+	if (name=="playback/speed_scale" || name=="speed_scale") { //bw compatibility
```

Comment:
The comment says "backwards compatibility", so it's not meant to be modified to the new name, it should actually be extended to support for newly deprecated name.